File: part1/Ex1class.java
public class Ex1class {

    private String nombre;
    private double salario;

    public Empleado (String nombre, double salario) {
        this.nombre = nombre;
        this.salario = salario;
    }

    public String getNombre() {
        return nombre;
    }

    public String toString() {
        return "Nombre: " + nombre + ", Salario: " + salario;
    }
}

File: part1/ex1class.ts
export class Empleado {

    private nombre: string;
    private salario: number;

    constructor(nombre: string, salario: number) {
        this.nombre = nombre;
        this.salario = salario;
    }

    getNombre(): string {
        return this.nombre;
    }

    toString(): string {
        return "Nombre:" + this.nombre + ", Salario: " + this.salario;
    }
}